src/main/resources/db/migration/V1_1__shedlock.sql

```diff
@@ -5,5 +5,3 @@ CREATE TABLE SHEDLOCK
     locked_at  TIMESTAMP(3) NULL,
     locked_by  VARCHAR(255) NOT NULL
 );
-
-GRANT SELECT, INSERT, UPDATE, DELETE ON SHEDLOCK TO ${flyway:user};
```

Comment:
This should do it, as the user flyway uses is the same as the application and it owns the schema